inc/cxplat_winkernel.h

```diff
@@ -603,6 +606,38 @@ CxPlatThreadCreate(
 typedef ULONG_PTR CXPLAT_THREAD_ID;
 #define CxPlatCurThreadID() ((CXPLAT_THREAD_ID)PsGetCurrentThreadId())
 
+inline
+BOOLEAN
+CxPlatSetThreadNodeAffinity(
+    LONG NodeAffinity
```

Comment:
Let's not use Windows style all caps types like `LONG`.
```suggestion
    uint32_t NodeAffinity
```